women-in-tech-resources/resources-list.md

```diff
@@ -69,6 +69,7 @@ If this is your first time looking at the Women in Tech Resources List, spend 30
 ### Scholarships:
 - [SWE Scholarship](http://societyofwomenengineers.swe.org/swe-scholarships)
 - [Palantir Scholarship](https://www.palantir.com/students/scholarship/wit-north-america/)
+-Venkat Panchapakesan Memorial Scholarship(https://buildyourfuture.withgoogle.com/scholarships/venkat-panchapakesan-memorial-scholarship/#!?detail-content-tabby_activeEl=overview)
```

Comment:
@Kumaranupam2020 Put `Venkat Panchapakesan Memorial Scholarship` in square bracket like this [Venkat Panchapakesan Memorial Scholarship].